### PR TITLE
Add Slice Thermistor to table header file

### PR DESCRIPTION
**Pull request by jonmason**
Add a table entry for the Slice Engineering 450c thermistor.  This
allows for easy modification of the config file when adding on to a
Prusa i3.  The table entry is taken from Chris Riley's Youtube Channel
video on this exact issue.

### CODE REVIEW

**Review by radimkohout — APPROVED** [review]

**Comment by 3d-gussner** [comment]
I am not sure if Prusa will add this thermistor table, but you can have a look at https://github.com/Zaribo/Zaribo-Firmware/tree/Zaribo-MK3/Firmware
and
https://github.com/vertigo235/Prusa-Firmware

These have that included
- as the Zaribo can be bought with the Slice Engineering thermistor
- as it is a community build supporting lot of different mods like Bear and others.

File: Firmware/variants/1_75mm_MK25-RAMBo10a-E3Dv6full.h [hunk 1]
```diff
@@ -439,6 +439,8 @@
 // 148 is E3D Pt100 with 4k7 pullup and no PT100 Amplifier on a MiniRambo 1.3a
 // 247 is Pt100 with 4k7 pullup and PT100 Amplifier
 // 110 is Pt100 with 1k pullup (non standard)
+// 800 is Slice 450c thermistor
+
```
Comment (wavexx) [inline Firmware/variants/1_75mm_MK25-RAMBo10a-E3Dv6full.h:443]: Extra empty line here?

**Comment by wavexx** [comment]
I have no complaints about this. The new table changes nothing unless you customize the variant yourself (and the patch is clean!)

**Comment by rgooler** [comment]
Looks like this PR was reviewed months ago and never merged? It would greatly simplify keeping firmware up to date for those of us with mosquito hotends on our prusas.

**Comment by 3d-gussner** [comment]
@rgooler Try https://github.com/Caribou3d/Caribou-Firmware which has support for some mods.

**Pull request closed without merging**